app/assets/stylesheets/components.scss

```diff
@@ -1575,11 +1597,15 @@ button.active i.fa-retweet {
   }
 
   .emoji-search-wrapper + .emoji-categories-wrapper {
-    top: 83px;
+    top: 93px;
   }
 
   .emoji-row .emoji:hover {
     background: lighten($color2, 3%);
+
+    img, svg {
+      transform: translateZ(0) scale(1.2);
```

Comment:
Hmm. I think I was misinformed about the performance implications - I believed that `translateZ(0)` activates GPU to make animations smoother (and I am planning to add a bouncy animation to this at some point.